parley/src/tests/utils/renderer.rs

```diff
@@ -0,0 +1,217 @@
+// Copyright 2024 the Parley Authors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! A simple example that lays out some text using Parley, extracts outlines using Skrifa and
+//! then paints those outlines using Tiny-Skia.
+//!
+//! Note: Emoji rendering is not currently implemented in this example. See the swash example
+//! if you need emoji rendering.
```

Comment:
I am slightly worried by the code duplication with the existing example. However, I don't think there's a good way for the code to be shared, unfortunately.

Reply:
I think that reasonable sharing of the code would lead to a less readable example. I also do not think that there is a good way for remove this duplication.